### DISCUSSION TITLE
Proposal for rebranding and renaming of the framework

### DISCUSSION THREAD

**Discussion by MagicSpins** [dotnet/orleans#8239] [Ideas]
The search output of materials on technology often contains content related to the city of New Orleans. Can at least become a name Orleans.NET ?

**Comment by JorgeCandeias** [comment]
I never have any issues when searching for the proper product name "Microsoft Orleans". The first results I get are always from Microsoft Learn or relevant videos from YouTube. Same as searching for "T-SQL" instead of just "SQL" to disambiguate between products.

**Reply by jeremylcarter** [reply]
Never been an issue for me either. Infact would do more damage Search/SEO wise to rename because a decade of blogs and articles are now using an old name - now you have to search for 2 things - the old name and the new name.
